dns-zone.tf

```diff
@@ -0,0 +1,12 @@
+data "azurerm_dns_zone" "play" {
+  name                = "us-hpccsystems-dev.azure.lnrsg.io"
+  resource_group_name = "app-dns-prod-eastus2"
+}
+
+resource "azurerm_dns_a_record" "play-record_set" {
+  name = "play-test"
```

Comment:
This should not be hardcoded.  It could be a variable with a computed default value based on var.product_name.

Reply:
> This should not be hardcoded. It could be a variable with a computed default value based on var.product_name.

Now, there is nothig is the "azurerm_dns_a_record" block that is hardcoded.